va/validation-authority.go

```diff
@@ -260,7 +261,9 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 		}
 	}
 
-	body, err := ioutil.ReadAll(httpResponse.Body)
+	// payload should be ~87 bytes, give a little leeway
```

Comment:
Capitalize, and add explanation: "allow for a bit of whitespace after the KeyAuthorization."

Also, let's make N a const and error out ("response too long") if `len(body) == maxLength`